lib/light-service/context/key_verifier.rb

```diff
@@ -2,9 +2,9 @@ module LightService; class Context
   class KeyVerifier
     attr_reader :context, :action
 
-    def initialize(context)
+    def initialize(context, action)
       @context = context
-      @action = context.current_action
+      @action = action
```

Comment:
I suppose so. I wasn't sure if it was used anywhere else (I didn't check though).